requirements/dev.in

```diff
@@ -1,5 +1,4 @@
-flask
```

Comment:
This was creating conflicting deps since `kytos` was supposed to install it, and it was bringing conflicting underlying dependencies with `click`.

Reply:
agreed!